webanno-doc/src/main/resources/META-INF/asciidoc/admin-guide/settings_general.adoc

```diff
@@ -43,5 +43,15 @@
 | Whether regular users can access their own profile to change their password and other profile information. This setting has no effect if WebAnno is running in pre-authentication mode.
 | false
 | true
+
+| userSelection.hideUsers
+| Whether the system's usernames will be shown into the dropdown list at the Users tab. In order for the Project manager to add a user into an annotation project she has to write into the same text Field the annotator’s full username. When there is a full match this username (and only this) is displayed in the dropdown list where the Project Manager can select it and add it to her project.
+| false
+| true
+
+| userSelection.minLengthCharacters
```

Comment:
I don't quite understand what this setting is for. I though the idea was that if `hideUsers` is true, the manager has to enter the entire name of the user to add into the field. But why is the min-length then needed. The dropdown should only ever show either nothing or the username entered by the user if there is a user with that name?